src/cache/index.test.slow.ts

```diff
@@ -1,13 +1,14 @@
-import { describe } from "node:test";
-import { expect, it } from "vitest";
+import { expect, it, describe } from "vitest";
 import { lightBlockCache } from ".";
 
+function delay(ms = 3000) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
```

Comment:
Was doing something that led me to pull this out into a function but ended up not needing to do this after all. Keeping the code cause it does the same thing, though.